AK/ReverseIterator.h

```diff
@@ -87,4 +87,18 @@ class SimpleReverseIterator {
     int m_index;
 };
 
+template<typename Container>
+struct ReverseWrapper {
+    Container& container;
+};
+
+template<typename Container>
+auto begin(ReverseWrapper<Container> wrapper) { return wrapper.container.rbegin(); }
+
+template<typename Container>
+auto end(ReverseWrapper<Container> wrapper) { return wrapper.container.rend(); }
```

Comment:
Instead of putting these in the global namespace, can we just put them in `ReverseWrapper`?

Reply:
Done :)